react-components/src/apps/details/AppDetails.js

```diff
@@ -0,0 +1,248 @@
+            });
+        }
+    }
+
+    render() {
+        const {details, searchRegexPattern, baseDebugId, intl, classes} = this.props;
+        const {loading, dialogOpen} = this.state;
+        const isExternal = details.app_type.toUpperCase() === constants.EXTERNAL_APP.toUpperCase();
+        const showAppURL = details.is_public || isExternal;
+        const {average, user, total} = details.rating;
+
+        if (details) {
+            return (
+                <React.Fragment>
+                    <Paper id={baseDebugId} style={{padding: 5, fontSize: 11}}>
+                        {loading &&
+                        <CircularProgress size={30} className={classes.loadingStyle} thickness={7}/>
+                        }
+                    <Grid container spacing={24} style={{paddingLeft: 5}}>
+                        <Grid item xs={12}>
+                            <Favorite baseDebugId={baseDebugId}
+                                      details={details}
+                                      isExternal={isExternal}
+                                      classes={classes}
+                                      onFavoriteClick={this.onFavoriteClick}/>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <b>{getMessage("descriptionLabel")}:</b>
+                            <Highlighter
+                                search={searchRegexPattern}>{details.description}</Highlighter>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <Typography variant="h6">
+                                {getMessage("detailsLabel")}
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <b>{getMessage("publishedOn")}</b> {formatDate(details.integration_date)}
+                        </Grid>
+                        <Grid item xs={12}>
+                            <b>{getMessage("integratorName")}</b>
+                            <Highlighter
+                                search={searchRegexPattern}>{details.integrator_name}
+                            </Highlighter>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <b>{getMessage("integratorEmail")}</b>
+                            <Highlighter search={searchRegexPattern}>
+                                {details.integrator_email}
+                            </Highlighter>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <b>{getMessage("detailsRatingLbl")} </b>
+                            <Rating
+                                placeholderRating={average}
+                                emptySymbol={<img src={whitestar} className="icon" alt="white star"/>}
+                                fullSymbol={<img src={goldstar} className="icon" alt="gold star"/>}
```

Comment:
Are these the same icons from GWT/GXT?  If so, I wonder if maybe Mariah could make up some newer looking stars for us to use?  Also, I'm not sure if we're reusing the same open/close book icons for trees as well, but maybe those could be refreshed too.  Just an idea - not necessary.

Reply:
Sounds good. I will get in touch with her...